analyzer/codechecker_analyzer/analyzers/clangtidy/config_handler.py

```diff
@@ -27,7 +27,8 @@ def set_checker_enabled(self, checker_name, enabled=True):
         """
         Enable checker, keep description if already set.
         """
-        if checker_name.startswith("Wno-") or checker_name.startswith("W"):
+        if checker_name.startswith('W') or \
```

Comment:
You removed `Wno-` from this condition. Isn't it change the behaviour of the analyze command?
For example if I give `-e Wno-unused-value` or `-d Wno-unused-value` to this command?

Reply:
Checking `Wno` and `W` prefixes is equivalent with checking `W` prefix.